egs/wsj/s5/steps/score_kaldi.sh

```diff
@@ -137,9 +137,89 @@ if [ $stage -le 1 ]; then
   fi
 fi
 
+if $cer && [ $stage -le 2 ] ; then
```

Comment:
My feeling was that would need a lot of careful checking to keep things between score_kaldi.sh and score_kaldi_cer.sh, as it relies on a couple of the variables being set and the files being created and so on. I think I'll close it (for now).